arrow-annotations/src/main/java/arrow/synthetic.kt

```diff
@@ -31,4 +31,5 @@ package arrow
   AnnotationTarget.TYPEALIAS
 )
 @MustBeDocumented
+@Deprecated("This annotation is deprecated along with the deprecation of the Arrow-meta kapt implementation.")
```

Comment:
This one is actually used by Arrow Meta (upcoming) not the kapt version.
```suggestion
```

Reply:
Oh okay, I missunderstood you before then 😅 Thanks 😉